bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -2257,14 +2258,18 @@ public void setBackground(Color color) {
 }
 
 private void setCurrentDeviceZoom(int newZoomFactor) {
```

Comment:
This method does not set the current device zoom anymore, but the native zoom, so please rename it.